test/seedu/common/UtilsTest.java

```diff
@@ -0,0 +1,42 @@
+package common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+import seedu.addressbook.common.Utils;
+
+public class UtilsTest {
+
+    @Test
+    public void testIsAnyNull() {
```

Comment:
This is not the standard to name test cases. It should be methodTesting_ScenarioTesting_expectedReturnValue.

i.e. isAnyNull_NullArgs_returnFalse